Modules/Loadable/Markups/VTKWidgets/vtkSlicerAngleRepresentation3D.cxx

```diff
@@ -69,6 +69,27 @@ vtkSlicerAngleRepresentation3D::vtkSlicerAngleRepresentation3D()
   this->ArcActor->SetMapper(this->ArcMapper);
   this->ArcActor->SetProperty(this->GetControlPointsPipeline(Unselected)->Property);
 
+  this->LineOccludedMapper = vtkSmartPointer<vtkPolyDataMapper>::New();
+  this->LineOccludedMapper->SetInputConnection(this->TubeFilter->GetOutputPort());
+  this->LineOccludedMapper->SetRelativeCoincidentTopologyLineOffsetParameters(0, -25000);
```

Comment:
Add explanation to magic number -25000. Maybe even better: define a helper function in a base class to set these offset parameters.